Lib/fontTools/feaLib/ast.py

```diff
@@ -112,7 +113,9 @@ class Element(object):
     """A base class representing "something" in a feature file."""
 
     def __init__(self, location=None):
-        #: location of this element - tuple of ``(filename, line, column)``
+        #: location of this element as a `FeatureLibLocation` object.
```

Comment:
I think this special comment `#:` should be right before the `self.something = ...` assignement it is referring to, for sphinx to treat it as such, so i'd put it right after the if statement. Or maybe it works already?